docs/streams/quickstart.html

```diff
@@ -172,27 +173,85 @@
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input
+all streams lead to kafka
+</pre>
+
 <p>
-with the following output data being printed to the console:
+This message will be processed by the Wordcount streams application and the following output data will be written to the <b>streams-wordcount-output</b> topic and printed by the console consumer:
 </p>
 
 <pre class="brush: bash;">
-all     1
-lead    1
-to      1
-hello   1
-streams 2
-join    1
-kafka   3
-summit  1
+&gt; bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 ..
+all	    1
+streams	1
+lead	1
+to	    1
+kafka	1
 </pre>
 
 <p>
-Here, the first column is the Kafka message key in <code>java.lang.String</code> format, and the second column is the message value in <code>java.lang.Long</code> format.
-Note that the output is actually a continuous stream of updates, where each data record (i.e. each line in the original output above) is
-an updated count of a single word, aka record key such as "kafka". For multiple records with the same key, each later record is an update of the previous one.
+Here, the first column is the Kafka message key in <code>java.lang.String</code> format, and the second column is the message value in <code>java.lang.Long</code> format indicating the current count of the key.
 </p>
 
+Now let's continue write one more messages with the console producer into the input topic <b>streams-file-input</b>:
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input
+all streams lead to kafka
+hello kafka streams
+</pre>
+
+And more output data will be printed by the console consumer:
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 ..
+all	    1
+streams	1
+lead	1
+to	    1
+kafka	1
+hello	1
+kafka	2
+streams	2
+</pre>
+
+Here the last printed lines <b>kafka 2</b> and <b>streams 2</b> indicate updates to the keys <b>kafka</b> and <b>streams</b> whose counts has been incremented from <b>1</b>
+to <b>2</b>. And with more messages written with the console producer to the input topic <b>streams-file-input</b>:
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input
+all streams lead to kafka
+hello kafka streams
+join kafka summit
+</pre>
+
+You can observe additional messages added to <b>streams-wordcount-output</b> topic, reflecting more updated word counts:
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-consumer.sh --bootstrap-server localhost:9092 ..
+all	    1
+streams	1
+lead	1
+to	    1
+kafka	1
+hello	1
+kafka	2
+streams	2
+join	1
+kafka	3
+summit	1
+</pre>
+
+As one can see, outputs of the Wordcount streams application is actually a continuous stream of updates, where each data record (i.e. each line in the original output above) is
```

Comment:
We don't say "streams application", do we?

It's either a "Kafka Streams application" or a "streaming application" or just "application", depending on context.